pkg/operator/operator.go

```diff
@@ -340,6 +340,18 @@ func New(
 	})
 	o.informers = append(o.informers, informer)
 
+	informer = cache.NewSharedIndexInformer(
+		o.client.ClusterOperatorListWatch(ctx, "ingress"),
+		&configv1.ClusterOperator{}, resyncPeriod, cache.Indexers{},
+	)
+
+	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		UpdateFunc: func(_, newObj interface{}) {
+			o.handleEvent(newObj)
+		},
+	})
+	o.informers = append(o.informers, informer)
```

Comment:
Do we need an informer? If I understand correctly, the ingress operator is either present or not but an update of the ingress operator's status doesn't need to trigger a reconciliation from CMO.
In a sense, it's more a static parameter that CMO needs to initialize on startup?

Reply:
Yeah I wasn't sure about this. The [capabilities installer enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md) says that components that where disabled at install time can be enabled post-install, so technically we should be able to react to `ingress` appearing.
That being said, I'm not sure how relevant this is for this epic and whether is worth the additional reconciliation loops for everybody else.

Reply:
A more lightweight option would be to use only an AddFunc handler. This way CMO wouldn't trigger a reconciliation on every change to the ingress cluster operator

Reply:
Right, that is the only thing we're after here anyway. I'm still waiting on input from the requester. Will adjust or drop this commit accordingly.